MercadoPagoSDK/MercadoPagoSDK/Core/Utils.swift

```diff
@@ -346,6 +346,38 @@ internal class Utils {
         return ""
     }
 
+    class func maskFinder(dictID: String, forKey: String) -> [TextMaskFormater]? {
```

Comment:
Ya que estamos moviendo este código, sugiero un nombre mas autodocumentado:
```suggestion
    class func getMasks(inDictionary dictID: String, withKey key: String) -> [TextMaskFormater]? {
```